### PR TITLE
Added externally hosted package documentation blog post

### PR DESCRIPTION
**Pull request by daveverwer**

### CODE REVIEW

File: Content/posts/externally-hosted-package-documentation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+```
+version: 1
+external_links:
+  documentation: “https://example.com/package/documentation/”
+```
+
+Here’s an [example](https://github.com/groue/GRDB.swift/blob/master/.spi.yml) from [GRDB](https://swiftpackageindex.com/groue/GRDB.swift), who already opted into this new feature.
+
+In terms of what visitors to package pages see, it makes no difference where the documentation Lives. Visitors still see a single “Documentation” link in the package page sidebar.
```
Comment (finestructure) [inline Content/posts/externally-hosted-package-documentation.md:25]: Should we mention somewhere (perhaps here) that we now have a default link to internal docs that always tracks the latest release version or the default branch?

**Review by finestructure — APPROVED** [review]

File: Content/posts/externally-hosted-package-documentation.md (hunk 1 above)
Comment (daveverwer) [reply Content/posts/externally-hosted-package-documentation.md:25]: Good idea. I just pushed an update.

**Comment by daveverwer** [comment]
Updated with the standard documentation link and also removed the bit about scoring as it's too in the weeds.

File: Content/posts/externally-hosted-package-documentation.md [hunk 2]
```diff
@@ -24,4 +24,10 @@ Here’s an [example](https://github.com/groue/GRDB.swift/blob/master/.spi.yml)
 
 In terms of what visitors to package pages see, it makes no difference where the documentation Lives. Visitors still see a single “Documentation” link in the package page sidebar.
 
-The final announcement for today is that we now count the presence of documentation (either external or hosted by us) as contributing to our internal package score. We have bigger plans for how we’ll use this package score in the future, but right now, it’s used as part of how we order search results.
+We also created a universal documentation link for any package that has documentation either hosted by us or externally. For any package with documentation, navigate to:
+
+```
+https://swiftpackageindex.com/owner/repository/documentation
```
Comment (finestructure) [inline Content/posts/externally-hosted-package-documentation.md:30]:
```suggestion
https://swiftpackageindex.com/{owner}/{repository}/documentation
```

**Review by finestructure — APPROVED** [review]

**Pull request merged**